### DISCUSSION TITLE
Experimental show name

### DISCUSSION THREAD

**Discussion by msellstedt** [abrechts/Phoenix-ELN#21] [Feature Proposals]
I think it would have been good to be able to give each experiment a name of ones own choice. The first time I try a reaction I give it a unique experiment number. The second time I run the same reaction I give it the same experiment number, but with the addition of "_2" as suffix. I prefer this way of naming my reactions, and even if there is an underlying incremental reference name for all reactions, it would be nice if there also could be a searchable "show name" for each reaction that you could decide yourself.

**Comment by abrechts** [comment]
Sorry, a unique and predictable experiment-ID is at the core of every ELN for reliable experiment reference, also within work groups and organizations. Therefore you definitely can’t modify this according to your personal preferences.
 
But let’s take a step back. If your goal is to identify a group of repeated experiments, then the “Same Step” experiment list on the right-side application window panel might a different way to do this. It dynamically lists all experiments containing the same reaction as the current experiment (which is the same as repeated experiments), sorted by yield or scale. You can also use the “Connects” dialog for identifying repeated experiments by clicking a step in its reaction sequence scheme.
